assets/queries/k8s/memory_requests_not_defined/query.rego

```diff
@@ -0,0 +1,18 @@
+package Cx
+
+CxPolicy [ result ] {
+   metadata := input.document[i].metadata
+   spec := input.document[i].spec 
+   containers := spec.containers
+   requests := containers[c].resources.requests
```

Comment:
The query won't detect that these attributes, *resources* and *requests*, are not present. If you wish to enforce an attribute to existing at a lower level, you should guarantee the parent exists. I suggest to test these different edge cases and present different results for each.